elyra/pipeline/airflow/package_catalog_connector/airflow-package-catalog.json

```diff
@@ -56,11 +56,30 @@
         },
         "airflow_package_download_url": {
           "title": "Airflow package download URL",
-          "description": "URL where the Apache Airflow package wheel can be downloaded.",
+          "description": "URL where the Apache Airflow package wheel can be downloaded",
           "type": "string",
           "uihints": {
             "category": "Source"
           }
+        },
+        "auth_id": {
+          "title": "User Id",
+          "description": "User id that has read access for the specified URL resource",
+          "type": "string",
+          "uihints": {
+            "minLength": 1,
```

Comment:
I think `minLength` should be a direct attribute of `"auth_id"` (and `"auth_password"`) rather than a ui-hint - assuming you want validation to catch empty strings when defined.